scripts/package/package-debian.sh

```diff
@@ -115,10 +115,6 @@ parseargs(){
         echo "Provide the NuGet name of the targetted Shared Framework. Missing option '--framework-nuget-name'" && help
     fi
 
-    if [ -z "$PREVIOUS_VERSION_URL" ]; then
```

Comment:
I see this is commented out below.  What's the plan for enabling upgrade tests?  Do we have an issue for it?  If not, can you log one?